src/Features/Core/Portable/StackTraceExplorer/StackTraceAnalyzer.cs

```diff
@@ -29,16 +31,32 @@ internal static class StackTraceAnalyzer
         public static Task<StackTraceAnalysisResult> AnalyzeAsync(string callstack, CancellationToken cancellationToken)
         {
             var parsedFrames = Parse(callstack, cancellationToken);
-            return Task.FromResult(new StackTraceAnalysisResult(parsedFrames.ToImmutableArray()));
+            return Task.FromResult(new StackTraceAnalysisResult(callstack, parsedFrames.ToImmutableArray()));
         }
 
         private static IEnumerable<ParsedFrame> Parse(string callstack, CancellationToken cancellationToken)
         {
-            foreach (var line in SplitLines(callstack))
+            // if the callstack comes from ActivityLog.xml it has been
+            // encoding to be passed over HTTP. This should only decode 
+            // specific characters like "&gt;" and "&lt;" to their "normal"
+            // equivalents ">" and "<" so we can parse correctly
+            callstack = WebUtility.HtmlDecode(callstack);
```

Comment:
can this throw exceptions?  do we need try/catch/bail-gracefully with this?

Reply:
Not that I know of. It accepts null and empty values, and doesn't document any cases where an exception is expected to be thrown. https://docs.microsoft.com/en-us/dotnet/api/system.web.httputility.htmldecode?view=net-6.0